### PR TITLE
Np 45262 cristin import artistic details mapped to description

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java [hunk 1]
```diff
@@ -62,6 +63,7 @@ public class CristinMapper extends CristinMappingModule {
     public static final String UNIT_INSTITUTION_CODE = "UNIT";
     public static final ResourceOwner SIKT_OWNER = new CristinLocale("SIKT", "20754", "0", "0",
                                                                      "0").toResourceOwner();
+    private static final String NEW_LINE = "\n";
```
Comment (torarnet) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:66]: There also is System.lineSeparator() or something which uses the correct separator for each system (linux, windows etc) but im not sure if its needed here.

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java [hunk 2]
```diff
@@ -270,16 +272,24 @@ private Instant extractEntryLastModifiedDate() {
...
+            .withNpiSubjectHeading(extractNpiSubjectHeading())
+            .withAbstract(extractAbstract())
+            .withTags(extractTags())
+            .withAlternativeAbstracts(Collections.emptyMap())
+            .withDescription(extractDescription())
+            .build();
+    }
+
+    private String extractDescription() {
+        return Optional.ofNullable(cristinObject.getCristinArtisticProduction())
```
Comment (torarnet) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:289]: is CristinArtisticProduction the only one that uses description?

**Review by torarnet — COMMENTED** [review]
See comments. Will approve afterwards

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java (hunk 2 above)
Comment (anetteOlli) [reply cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:289]: So far yes, (and most likely to be so forever).

It all the fields from cristin that doesn't quite fit the NVA-datamodel. Not a design award this implementation, but it's either that or  dataloss.

**Review by torbjokv — APPROVED** [review]

**Review by torarnet — APPROVED** [review]

**Pull request merged**